sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilderTests.java

```diff
@@ -303,14 +304,33 @@ public void encryptedRegionLength(long regionLength) {
 
     @Test
     public void encryptedRegionLengthDefault() {
-        EncryptedBlobClient encryptedBlobClient = new EncryptedBlobClientBuilder(EncryptionVersion.V2)
+        EncryptedBlobClient encryptedBlobClient = new EncryptedBlobClientBuilder()
```

Comment:
Oh so for this one, I was suggesting to leave the version as `EncryptionVerson.V2` not remove the version all together. There should not be a diff for this line for this PR.